CMakeLists.txt

```diff
@@ -324,6 +329,15 @@ if(WIN32)
   endif()
 endif()
 
+if(USE_GLOO_WITH_OPENSSL AND LINUX)
+  find_package(OpenSSL REQUIRED)
```

Comment:
Made it OPENSSL_init_ssl for testing purposes to make sure that we have libssl-dev/openssl-develop installed on CI environments